manager/main.go

```diff
@@ -262,7 +262,10 @@ func run(namespace, metricsAddr, healthProbeAddr string, enableLeaderElection bo
 		// Initiate the Blueprint Controller
 		localMountPath := os.Getenv("LOCAL_CHARTS_DIR")
 		setupLog.Trace().Str("local charts dir", localMountPath).Msg("creating Blueprint controller")
-		blueprintController := app.NewBlueprintReconciler(mgr, "Blueprint", helm.NewHelmerImpl(localMountPath))
+		// isMultiCluster := !(enableApplicationController || !enablePlotterController) || clusterManager.IsMultiClusterSetup()
+		// blueprintController := app.NewBlueprintReconciler(mgr, "Blueprint", helm.NewHelmerImpl(localMountPath),
+		// 	clusterManager.IsMultiClusterSetup())
+		blueprintController := app.NewBlueprintReconciler(mgr, "Blueprint", helm.NewHelmerImpl(localMountPath), true)
```

Comment:
Why is it always true?

Reply:
In order to check if the multicluster deployment is enabled we can use the function `clusterManager.IsMultiClusterSetup()` which returns true when `razee` is deployed.

Reply:
I don't see the proposed change (using clusterManager.IsMultiClusterSetup() instead of true)